policyengine_us/parameters/gov/states/wa/tax/income/capital_gains/deductions/charitable/cap.yaml

```diff
@@ -1,12 +1,15 @@
 description: Washington capital gains charitable contribution cap.
 metadata:
   label: Washington capital gains charitable contribution cap
-  name: wa_ltcg_charitable_contributions_cap
   unit: currency-USD
   period: year
   reference:
-    - title: "RCW 82.87.080(2) Charitable donation deduction"
+    - title: RCW 82.87.080(2) Charitable donation deduction
       href: https://app.leg.wa.gov/RCW/default.aspx?cite=82.87.080
+    - title: Capital gains tax | Washington Department of Revenue
+      href: https://dor.wa.gov/taxes-rates/other-taxes/capital-gains-tax
+    - title: 2021 Tax Legislation | Washington Department of Revenue
+      href: https://dor.wa.gov/forms-publications/publications-subject/tax-topics/2021-tax-legislation
 values:
-  0000-01-01: 0
-  2022-01-01: 100_000
+  2021-01-01: 100_000
```

Comment:
is it uprated with inflation?

Reply:
same for all yaml

Reply:
https://lawfilesext.leg.wa.gov/biennium/2021-22/Pdf/Bills/Session%20Laws/Senate/5096-S.SL.pdf#page=17

> Beginning December 2023 and each December thereafter, the department must adjust the applicable amounts by multiplying the current applicable amounts by one plus the percentage by which the most current consumer price index available on December 1st of the current year exceeds the consumer price index for the prior 12-month period, and rounding the result to the nearest $1,000.

Reply:
Please add uprating metadata with rounding

Reply:
for all yaml